working/0723-static-extensions/feature-specification.md

```diff
@@ -1,35 +1,88 @@
 # Extensions with Static Capabilities
 
-Author: Erik Ernst
+Authors: Erik Ernst, Leaf Petersen
 
 Status: Draft
 
-Version: 1.2
+Version: 1.3
 
 Experiment flag: static-extensions
 
 This document specifies extensions with static capabilities. This is a
 feature that supports the addition of static members and/or constructors to
 an existing declaration that can have such members, based on a
-generalization of the features offered by `extension` declarations.
+generalization of the features offered by extension declarations.
 
 ## Introduction
 
-A feature like extensions with static capabilities was requested already
-several years ago in [language issue #723][issue 723], and elsewhere.
+An extension declaration in Dart can already (before the addition of
+the feature specified here) declare static members, but not
+constructors.  Members so declared are only accessible via prefixing
+with the extension name.
+
+```dart
+extension Numbers on int {
+  static int one = 1;
+  static int two = 2;
+}
+void main() {
+  // Static members declared in an extension can be accessed by prefixing
+  // with the extension name.
```

Comment:
Several of these source code snippets have comments which are cut off when the document is printed on A4 paper. This does happen on screen, too, with a smallish window. So I'm suggesting changes to make those comments so much shorter that it doesn't happen (just in case I'm not the only person on this planet who could get the idea that it would be nice to read this on paper ;-).

```suggestion
  // Static members declared in an extension can be accessed by
  // prefixing with the extension name.
```